src/packages/@ncigdc/modern_components/ClinicalCard/ClinicalCard.js

```diff
@@ -117,7 +117,10 @@ export default compose(
                     diagnoses.length > 1
                       ? diagnoses.map(x => (
                           <p key={x.node.diagnosis_id}>
-                            {truncate(x.node.diagnosis_id, 8)}
+                            {_.truncate(x.node.diagnosis_id, {
+                              length: 9,
+                              omission: '…',
```

Comment:
you can drop this key. the default is `...`

Reply:
Ok cool. I was wondering if I should bother being explicit.